### PR TITLE
Add missing colon in prune stats output and realign the fields

### LINKED ISSUE #3617: Missing colon in prune output
Output of `restic version`
--------------------------

restic 0.12.1 compiled with go1.16.6 on freebsd/amd64

How did you run restic exactly?
-------------------------------

restic prune

What backend/server/service did you use to store the repository?
----------------------------------------------------------------

sftp

Expected behavior
-----------------

to repack:          907 blobs / 3.743 MiB  
this removes:        191 blobs / 1002.958 KiB
to delete:          212 blobs / 90.811 MiB                                                                                                             
total prune:        403 blobs / 91.791 MiB
remaining:        32232 blobs / 6.252 GiB
unused size after prune: 271.040 MiB (4.23% of remaining size)

Actual behavior
---------------

to repack:          907 blobs / 3.743 MiB  
this removes        191 blobs / 1002.958 KiB
to delete:          212 blobs / 90.811 MiB                                                                                                             
total prune:        403 blobs / 91.791 MiB
remaining:        32232 blobs / 6.252 GiB
unused size after prune: 271.040 MiB (4.23% of remaining size)

Steps to reproduce the behavior
-------------------------------

Run restic prune and view the output. It looks like "this removes" should have a colon after it so that it matches the other lines.

Do you have any idea what may have caused this?
-----------------------------------------------

Accidently left off

Do you have an idea how to solve the issue?
-------------------------------------------

Add the colon into line 465 of cmd/restic/cmd_prune.go

Did restic help you today? Did it make you happy in any way?
------------------------------------------------------------

Restic is an excellent tool. I use it for multiple backups to B2, AWS-S3, and SFTP and it has never let me down. Thank you very much for it!

### PR DESCRIPTION
**Pull request by ghost**
What does this PR change? What problem does it solve?
-----------------------------------------------------

This PR adds a missing colon in the "this removes" field of the prune output and realigns the other fields to match.

Was the change previously discussed in an issue or on the forum?
----------------------------------------------------------------

Closes #3617

Checklist
---------

- [ ] I have read the [contribution guidelines](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#providing-patches).
- [X] I have [enabled maintainer edits](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added tests for all code changes.
- [ ] I have added documentation for relevant changes (in the manual).
- [ ] There's a new file in `changelog/unreleased/` that describes the changes for our users (see [template](https://github.com/restic/restic/blob/master/changelog/TEMPLATE)).
- [ ] I have run `gofmt` on the code in all commits.
- [ ] All commit messages are formatted in the same style as [the other commits in the repo](https://github.com/restic/restic/blob/master/CONTRIBUTING.md#git-commits).
- [X] I'm done! This pull request is ready for review.

### CODE REVIEW

**Comment by rawtaz** [comment]
Thanks. Can you also add three spaces before the `%10d` in lines 478-480, and then squash your commit(s)?

**Comment by mattxtaz** [comment]
That has been done. Tested and the alignment looks OK to me.

File: cmd/restic/cmd_prune.go [hunk 1]
```diff
@@ -475,9 +475,9 @@ func prune(opts PruneOptions, gopts GlobalOptions, repo restic.Repository, usedB
...
 	Verboseff("unused packs:       %10d\n\n", stats.packs.unused)
 
-	Verboseff("to keep:   %10d packs\n", stats.packs.keep)
-	Verboseff("to repack: %10d packs\n", len(repackPacks))
-	Verboseff("to delete: %10d packs\n", len(removePacks))
+	Verboseff("to keep:      %10d packs\n", stats.packs.keep)
+	Verboseff("to repack:    %10d packs\n", len(repackPacks))
+	Verboseff("to delete:    %10d packs\n", len(removePacks))
 	if len(removePacksFirst) > 0 {
 		Verboseff("to delete: %10d unreferenced packs\n\n", len(removePacksFirst))
```
Comment (rawtaz) [inline cmd/restic/cmd_prune.go:482]: Need three additional spaces here too :)

**Comment by rawtaz** [comment]
Sorry about the nitpicking, but once you added those last three spaces I commented on above, there's also the `060_forget.rst` file that needs to be updated as it contains example output from a `prune` run. Basically just add a colon on line 140, add spaces if you want (noone will notice anyway). Then we can merge this.

File: cmd/restic/cmd_prune.go (hunk 1 above)
Comment (mattxtaz) [reply cmd/restic/cmd_prune.go:482]: I'm confused. I already added 3 spaces there, rebased it to squash it up, and force pushed it. The three spaces show up when I look at the commit?

**Comment by mattxtaz** [comment]
Doc file updated as well. Rebased/Pushed. H

File: cmd/restic/cmd_prune.go (hunk 1 above)
Comment (MichaelEischer) [reply cmd/restic/cmd_prune.go:482]: I think this comment refers to this line: "to delete: %10d unreferenced packs\n\n". That one belongs to the same group of output values.

Comment (mattxtaz) [reply cmd/restic/cmd_prune.go]: That makes sense. I've edited that line as well. Thanks!

**Review by rawtaz — APPROVED** [review]
Looks perfect!

**Pull request merged**